### ISSUE TITLE
Results of the MRMC analysis depends on the order of "ModalityID" in the FROC dataset.

### ISSUE THREAD

**Issue by technOslerphile** [dpc10ster/RJafroc#89]
[frocCr.xlsx](https://github.com/dpc10ster/RJafroc/files/11615312/frocCr.xlsx)
It looks like the results of StSignificanceTesting depends on the order of ModalityID in TP and FP sheets. 
```
library("RJafroc")
library("readxl")
library("xlsx")
x <- DfReadDataFile(fileName = "frocCr.xlsx", newExcelFileFormat = TRUE)
ret <- StSignificanceTesting(x, FOM = "AFROC", method = "OR", analysisOption = "RRRC")
ret$RRRC$ciDiffTrt


truth <- read_xlsx("frocCr.xlsx", sheet = "TRUTH")
tp <- read_xlsx("frocCr.xlsx", sheet = "TP")
fp <- read_xlsx("frocCr.xlsx", sheet = "FP")

write.xlsx(as.data.frame(tp), file = "froc1.xlsx",
                 sheetName = "TP", append = FALSE, row.names = F, showNA = F)
write.xlsx(as.data.frame(fp), file = "froc1.xlsx", 
           sheetName="FP", append=TRUE, row.names = F)
write.xlsx(as.data.frame(truth), file = "froc1.xlsx",
           sheetName="TRUTH", append=TRUE, row.names = F)
x <- DfReadDataFile(fileName = "froc1.xlsx", newExcelFileFormat = TRUE)
ret1 <- StSignificanceTesting(x, FOM = "AFROC", method = "OR", analysisOption = "RRRC")

#reorder tp and fp

tp <- tp %>% arrange(desc(tp))
fp <- fp %>% arrange(desc(fp))
head(tp)
head(fp)

write.xlsx(as.data.frame(tp), file = "froc1.xlsx",
           sheetName = "TP", append = FALSE, row.names = F, showNA = F)
write.xlsx(as.data.frame(fp), file = "froc1.xlsx", 
           sheetName="FP", append=TRUE, row.names = F)
write.xlsx(as.data.frame(truth), file = "froc1.xlsx",
           sheetName="TRUTH", append=TRUE, row.names = F)
x <- DfReadDataFile(fileName = "froc1.xlsx", newExcelFileFormat = TRUE)
ret2 <- StSignificanceTesting(x, FOM = "AFROC", method = "OR", analysisOption = "RRRC")

ret1$RRRC
ret2$RRRC
```

<img width="299" alt="image" src="https://github.com/dpc10ster/RJafroc/assets/8592080/969d82d5-b9fc-4f0c-be4e-17a5e45985db">

As you can see from the above screenshot, the trt1-trt0 and trt0-trt1 shows the same result. This is not intuitive and can be misleading with opposite inferences.

**Comment by dpc10ster** [comment]
Thanks for posting this; I am looking into a fix; Dev

**Comment by dpc10ster** [comment]
Fixed in da5b408633b4dde73fc58161db995a9dde8facff

* Program expected `TP` and `FP` worksheets to obey "good" ordering.
* If shuffled then unexpected results can occur.
* Added a sorting step to both worksheets.

**Comment by technOslerphile** [comment]
Thanks @dpc10ster . How will this new fix be made available in the R package? 
Will installing RJafroc using `devtools::install_github` be sufficient? I suppose the CRAN version is still the old version.
